mediator/med_fraction_mod.F90

```diff
@@ -682,7 +682,7 @@ subroutine med_fraction_set(gcomp, rc)
 
        call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_ifrac', Si_ifrac, rc=rc)
        if (ChkErr(rc,__LINE__,u_FILE_u)) return
-       call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_imask', Si_imask, rc=rc)
+       call FB_getFldPtr(is_local%wrap%FBImp(compice,compice) , 'Si_imask' , Si_imask, rc=rc)
```

Comment:
remove space?

Reply:
I will double check the several white space changes. They should all be in the merge to ESCOMP and will go into our emc/develop w/ this PR.